test/unit/adapters/http.js

```diff
@@ -660,6 +660,24 @@ describe('supports http with nodejs', function () {
       });
     });
   });
+
+  it('should allow methods', function (done) {
```

Comment:
What's the meaning of this added case? Looks like it wasn't mentioned in the pull request description.